packages/core-typings/src/Apps.ts

```diff
@@ -45,10 +66,11 @@ export type App = {
 	version: string;
```

Comment:
```
detailedDescription: IDetailedDescription;
detailedChangelog: IDetailedChangelog;
```
These are missing, there were defined above and got lost.

Reply:
detailedDescription was defined on the spot. 
detailedChangelog wasn't being used and, consequently, not defined, I'll be defining it on the next commit for future endeavors.